### PR TITLE
View points

### PR DESCRIPTION
**Pull request by isamitre**
User (which I'm getting from the database rn with a hardcoded id) can view their points program page, complete with a PointsBar and UserEventsTable component
image

### CODE REVIEW

File: util/hooks.js [hunk 1]
```diff
@@ -1,40 +1,83 @@
...
-
-      if (error != errorArray[errorArray.length - 1]) {
-        errorString += "\n";
-      }
-    })
-
-    Alert.alert(errorString);
-  }
-};
+<<<<<<< HEAD
```
Comment (juansuh) [inline util/hooks.js]: you left git artifacts in your code, this isnt actually a resolved merge conflict

File: util/hooks.js [hunk 2]
```diff
@@ -1,40 +1,83 @@
...
+      if (error != errorArray[errorArray.length - 1]) {
+        errorString += "\n";
+      }
+    })
+
+    Alert.alert(errorString);
+  }
+};
+=======
+import { useState } from "react";
```
Comment (juansuh) [inline util/hooks.js]: see, there was a merge conflict and u just hit save

File: components/UserEventsTable.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
+import React, { Component } from "react";
+import { Alert, Button, View, Text, StyleSheet } from "react-native";
+import { Table, Row, Rows, Cell } from "react-native-table-component";
+
+import moment from "moment";
```
Comment (juansuh) [inline components/UserEventsTable.js]: this is a super fat library, please take this out

Comment (juansuh) [reply components/UserEventsTable.js]: they aint even in active development anymore

File: util/hooks.js (hunk 1 above)
Comment (isamitre) [reply util/hooks.js]: fixed

File: yarn.lock [hunk 4]
```diff
@@ -0,0 +1,14215 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+"@apollo/react-common@^3.1.4":
```
Comment (juansuh) [inline yarn.lock]: giant yarn.lock file added, did you try using yarn? this is an npm project so delete this since we already have a package-lock.kson

**Pull request merged**